test/codegen-generated-classes-test/src/main/resources/codegen-resources/eventstreams/service-2.json

```diff
@@ -75,7 +75,7 @@
     "EventStream": {
       "type": "structure",
       "members": {
-        "EventOne": {
+        "TheEventOne": {
```

Comment:
Note: legacy uses event shape name, so this caused a clash because we generated two "visitEventOne" methods; one is because the non legacy event was called "eventOne", and the legacy customized one had a shape name of "EventOne"